examples/spire-sidecar/config/apps.python.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: apps
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: apps
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        identity_template: "true"
+        app: apps
+    spec:
+      initContainers:
+        - name: sidecar
+          securityContext:
+            # privilaged is needed to create socket and bundle files
+            privileged: true
+          image: tsidentity/tornjak-example-sidecar:v0.1
+          imagePullPolicy: Always
+          # use command to override the command of the image/dockerfile
+          command: ["python3", "/usr/local/bin/run-sidecar-python.py", "/usr/local/bin/inputfile.txt"]
+          env:
+          - name: SOCKETFILE
+            value: "/run/spire/sockets/agent.sock"
+          - name: ROLE
+            value: "dbrole"
+          - name: VAULT_ADDR
+            value: "http://tsi-vault-tsi-vault.space-x04-9d995c4a8c7c5f281ce13d5467ff6a94-0000.eu-de.containers.appdomain.cloud"
+          volumeMounts:
+            - name: spire-agent-socket
+              mountPath: /run/spire/sockets
+              readOnly: true
+            - name: db-config
+              mountPath: /run/db
+            - name: mount-inputfile
+              mountPath: /usr/local/bin/inputfile.txt
+              subPath: inputfile.txt
+      containers:
+        - image: tsidentity/tornjak-example-nodejs:v0.1
+          imagePullPolicy: Always
+          name: node
+          ports:
+            - containerPort: 8080
+          resources: {}
+          volumeMounts:
+          - mountPath: /usr/src/app/config.json
+            name: db-config
+            readOnly: true
+            subPath: config.json
+        - image: tsidentity/tornjak-example-python:v0.1
+          imagePullPolicy: Always
+          name: py
+          ports:
+            - containerPort: 5000
+          resources: {}
+          volumeMounts:
+          - mountPath: /app/config.ini
+            name: db-config
+            readOnly: true
+            subPath: config.ini
+      restartPolicy: Always
+      volumes:
+        - name: spire-agent-socket
+          hostPath:
+            path: /run/spire/sockets
+            type: Directory
+        - name: db-config
+          emptyDir: {}
+        - name: mount-inputfile
+          configMap:
+            name: path-to-inputfile
+status: {}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: apps
+spec:
+  type: NodePort
+  selector:
+    app: apps
+  ports:
+    - name: "8001"
+      port: 8001
+      targetPort: 8080
+    - name: "8000"
+      port: 8000
+      targetPort: 5000
+status:
+  loadBalancer: {}
+---
+apiVersion: v1
```

Comment:
We should move the ConfigMap to the front of the file. This might not always be guaranteed that completes before the Deployment, but it might help